dapp/src/components/buySell/ApproveButtonLogic.js

```diff
@@ -11,6 +11,14 @@ import ContractStore from 'stores/ContractStore'
 import analytics from 'utils/analytics'
 import { assetRootPath } from 'utils/image'
 import { useWeb3React } from '@web3-react/core'
+import Media from 'react-media'
```

Comment:
Can you use `withIsMobile` instead of `react-media` in order to be consistent. See example here: https://github.com/OriginProtocol/origin-dollar/blob/7074ecfac35b5f362c75afc35fffcce8e15aa63b/dapp/src/components/buySell/SwapHomepage.js#L26

Reply:
I tried with that, but it was not working... Something like {withIsMobile ? '' : ''}

Reply:
Maybe this example is better: https://github.com/OriginProtocol/origin-dollar/blob/master/dapp/src/components/buySell/BalanceHeader.js#L24

when you wrap a component in `withIsMobile` it is going to have a `isMobile` property that will communicate if page should render in mobile mode